resources/item_id_cache.json

```diff
@@ -0,0 +1,3 @@
+{
+
+}
```

Comment:
No need to have this and the block_id_cache.json file in the folder if they are not in use.